lib/ord.js

```diff
@@ -30,8 +32,13 @@ const initializeAppConfig = (csn) => {
     const modelKeys = Object.keys(csn.definitions);
     const apiEndpoints = new Set();
     const events = [];
+    let externalServiceKeys = [];
+    if (packageJson.cds && packageJson.cds.requires) {
```

Comment:
What will happen, when the information is provided through the cdsrc.json and not via package json?

Reply:
if you have configurations in both package.json and cdsrc.json then it is considering both the configurations and both the objects are there in the integrationDependencies array
 
So, which one should be omitted or given more priority?
 
 Consider the following case:
![Screenshot 2024-11-28 at 1 55 23 PM](https://github.com/user-attachments/assets/f9c3e796-7ff4-4b4f-b9dd-c759328d9425)

Reply:
@Dipto-at-sap ,  the comment was related to the issue @David-Kunz highlighted, you can have cds.requires in package.json and cdsrc.json the custom.ord.json is another aspect. 

With regards to the custom.ord.json, it must be consistent with the behavior described in https://github.com/cap-js/ord/blob/main/docs/ord.md. @zongqichen can support you in case of questions on that. Please reach out directly to him or @aramovic79 in his absence.

Reply:
This is related to David's comment. So, we'll discuss it in the call.